app/src/main/java/edu/uncc/inclass11/GradesFragment.java

```diff
@@ -189,6 +191,14 @@ void setCourse_number(String course_number) {
         void setCourse_name(String course_name) {
             TextView textView = view.findViewById(R.id.textViewCourseName);
             textView.setText(course_name);
+
+            ImageView imageViewDelete = view.findViewById(R.id.imageViewDelete);
+            imageViewDelete.setOnClickListener(view -> firebaseFirestore.collection("grades").document(course_name)
```

Comment:
This is not the correct path to a document. Take a look at [lines 100-104](https://github.com/kstanley-UNCC/Group22_InClass11/pull/2/files#diff-73ba99bbb8a2513b7508333ba2abad5216d7d8cdfd1970fc69a96d62bbda2e80R100-R104) to see the correct path used.